sync_sdk_docs.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+REMOTE_REPO_URL="https://github.com/cosmos/cosmos-sdk.git"
```

Comment:
can we add some comments to this file. Its easy to understand, comments would help document it a little better